book/from_flat_php_to_symfony2.rst

```diff
@@ -598,7 +598,7 @@ database and the Templating component to render a template and return a
     <ul>
         <?php foreach ($posts as $post): ?>
         <li>
-            <a href="<?php echo $view['router']->generate(
+            <a href="<?php echo $view['router']->path(
```

Comment:
no versionadded comment here, as I don't think those are usefull in this specific flat PHP to Symfony article.